docs/snowflake native app architecture docs (#3359)
dlt-hub/dlt: docs/website/docs/walkthroughs/run-in-snowflake/run-in-snowflake.md

---
title: Run dlt in Snowflake
description: Run dlt in Snowflake Native App
keywords: [snowflake, native app, dlt]
---

> dlt can run in or with Snowflake in a few different ways:

Reviewer: ```suggestion
You can run dlt within Snowflake AI Data Cloud in a few different ways.
```

## Ways to run dlt

1. **Snowflake Native App**  

>    Install the **Database Connector App** from the Snowflake Marketplace and run pipelines through the app UI.  

Reviewer: ```suggestion
   Install the **Database Connector App** from the Snowflake Marketplace and run pipelines through the built-in UI.  
```

2. **Snowflake Notebook**  
   Run a dlt pipeline directly inside a Snowflake Notebook for interactive development and testing.  

3. **Snowpark Container Services (SPCS)**  
   Package dlt into a container and run it on SPCS for in-Snowflake execution.  

4. **External runner with Snowflake as destination**  
   Run dlt outside Snowflake (your infra or a third party) and load into Snowflake as the destination.  

> ## Choosing an option

Reviewer: ```suggestion
## How to choose
```

---
title: Run dlt in Snowflake
description: Run dlt in Snowflake Native App
keywords: [snowflake, native app, dlt]
---

> 

Reviewer: Let's not leave it completely empty! Could you just add a paragraph that there are several different ways to run dlt directly on the Snowflake.
